spotseeker_server/management/commands/techloan/spotseeker.py

```diff
@@ -167,15 +169,25 @@ def _get_image_id(self, items: list, item_id) -> int:
             if item['id'] == item_id:
                 return item['images'][0]['id']
 
-    def _download_image(self, image_url, cte_type_id):
+    def _download_image(self, image_url, cte_type_id) -> str:
         try:
-            image, _ = urlretrieve(image_url)
-            return image
+            file_name, _ = urlretrieve(image_url)
```

Comment:
Should we even keep using urlretrieve? The main benefit is it's done in fewer lines but it doesn't work in all cases (apparently because of it's "user agent"), but the requests way of doing it has to be there anyway now.